client/additional-methods-setup/upe-preview-methods-selector/add-payment-methods-task.js

```diff
@@ -158,8 +158,7 @@ const AddPaymentMethodsTask = () => {
 	);
 
 	const completeActivation = ( itemId ) => {
-		paymentMethodsState[ itemId ] = true;
-		handlePaymentMethodChange( paymentMethodsState );
+		handlePaymentMethodChange( itemId, true );
```

Comment:
Would it be possible to rename `itemId` (e.g. `method`) - by the context, it seems to be a payment method in there, but I'm not 100% sure?

Reply:
Thanks for calling this out, I have renamed it to `method`  - 55ec5d7